R/sample_analysis/Assigning data in datashield

```diff
@@ -0,0 +1,34 @@
+#Assigning data in DataSHIELD
+
+#LOAD PACKAGES
+library(opal)
+library(dsBaseClient)
+library(dsStatsClient)
+library(dsGraphicsClient)
+library(dsModellingClient)
+library(datashieldclient)
```

Comment:
So only opal and datashieldclient --> the latter one will load the dsBaseClient as well

Reply:
@angelapinotdemoira You are not replying ;-D.

Reply:
@angelapinotdemoira Check: https://www.youtube.com/watch?v=YZxy7MUrsrM&t=4s. With sound ;-D.

Reply:
As we heard the datashieldclient is apparently deprecated so only load dsBaseClient.